tests/lib/naming_test.py

```diff
@@ -169,5 +171,123 @@ def testParseServiceFile(self):
     filedefs._ParseFile(data, 'services')
     self.assertEqual(filedefs.GetService('HTTP'), ['80/tcp'])
 
+
+class Naming_Characterization_Tests(unittest.TestCase):
```

Comment:
@ankenyr, do you think there's any value for users to see what method calls with the existing data would return?  Maybe I could recast this somehow so it's some form of documentation.

Re those tests calling other tests - I have literally no idea what I was thinking with this.  Thanks, removing.